### ISSUE TITLE
Suggestion: Filter Graveyard by torrents you downloaded

### ISSUE THREAD

**Issue by TomArrow** [HDInnovations/UNIT3D#502]
### Issue Type
- [ ] Bug Report
- [ ] Feature Idea
- [ X ] Enhancement
- [ ] Technical Discussion
- [ ] Question

It would be nice if torrents in the graveyard could be filtered by torrents you have downloaded yourself.

**Comment by singularity43** [comment]
I like this logic.

Will look into adding this since it's a simple filter/view change and shouldn't be disputed by anybody.

**Comment by HDVinnie** [comment]
Yeah this is a easy one. Additional filter checkbox that only shows graveyard torrents torrent_id's that are matching in a users torrent history.
